libs/features/mas/package.json

```diff
@@ -15,29 +15,29 @@
     "build": "npm run build --workspaces"
   },
   "dependencies": {
-    "@spectrum-web-components/action-button": "^0.43.0",
-    "@spectrum-web-components/action-menu": "^0.43.0",
-    "@spectrum-web-components/button": "^0.43.0",
-    "@spectrum-web-components/button-group": "^0.43.0",
-    "@spectrum-web-components/checkbox": "^0.43.0",
-    "@spectrum-web-components/dialog": "^0.43.0",
-    "@spectrum-web-components/icon": "^0.43.0",
-    "@spectrum-web-components/icons": "^0.43.0",
-    "@spectrum-web-components/icons-workflow": "^0.43.0",
-    "@spectrum-web-components/link": "^0.43.0",
-    "@spectrum-web-components/menu": "^0.43.0",
-    "@spectrum-web-components/overlay": "^0.43.0",
-    "@spectrum-web-components/popover": "^0.43.0",
-    "@spectrum-web-components/progress-circle": "^0.43.0",
-    "@spectrum-web-components/radio": "^0.43.0",
-    "@spectrum-web-components/reactive-controllers": "^0.43.0",
-    "@spectrum-web-components/search": "^0.43.0",
-    "@spectrum-web-components/sidenav": "0.42.1",
-    "@spectrum-web-components/split-button": "^0.43.0",
-    "@spectrum-web-components/tabs": "^0.43.0",
-    "@spectrum-web-components/theme": "^0.43.0",
-    "@spectrum-web-components/tooltip": "^0.43.0",
-    "@spectrum-web-components/tray": "^0.43.0",
+    "@spectrum-web-components/action-button": "^0.45.0",
+    "@spectrum-web-components/action-menu": "^0.45.0",
+    "@spectrum-web-components/button": "^0.45.0",
+    "@spectrum-web-components/button-group": "^0.45.0",
+    "@spectrum-web-components/checkbox": "^0.45.0",
+    "@spectrum-web-components/dialog": "^0.45.0",
+    "@spectrum-web-components/icon": "^0.45.0",
+    "@spectrum-web-components/icons": "^0.45.0",
+    "@spectrum-web-components/icons-workflow": "^0.45.0",
+    "@spectrum-web-components/link": "^0.45.0",
+    "@spectrum-web-components/menu": "^0.45.0",
+    "@spectrum-web-components/overlay": "^0.45.0",
+    "@spectrum-web-components/popover": "^0.45.0",
+    "@spectrum-web-components/progress-circle": "^0.45.0",
+    "@spectrum-web-components/radio": "^0.45.0",
+    "@spectrum-web-components/reactive-controllers": "^0.45.0",
+    "@spectrum-web-components/search": "^0.45.0",
+    "@spectrum-web-components/sidenav": "^0.45.0",
+    "@spectrum-web-components/split-button": "^0.45.0",
+    "@spectrum-web-components/tabs": "^0.45.0",
+    "@spectrum-web-components/theme": "^0.45.0",
+    "@spectrum-web-components/tooltip": "^0.45.0",
+    "@spectrum-web-components/tray": "^0.45.0",
```

Comment:
this deps should not be needed.
I think they have been copied mistakenly during mas -> milo move.
Can you remove these and try again?

Reply:
Looks like they are needed, without them it's not possible to build web-components